### PR TITLE
feat(component): add cell selection with a keyboard

### PR DESCRIPTION
**Pull request by bc-apostoliuk**
## What? Why?

1. Implement the selection of multiple cells in a row using Shift+arrow buttons.
2. Implement the selection of multiple cells in a row within a column by holding SHIFT and clicking on the first and the last cells in the selection.

## Screenshots/Screen Recordings

https://user-images.githubusercontent.com/84462142/226999149-65a2b457-abb1-4577-86af-8a2b7bf4e1e8.mov

## Testing/Proof
Locally + UT

### CODE REVIEW

File: packages/big-design/jest.config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = {
       statements: 95.73,
       branches: 86.97,
       functions: 97,
-      lines: 95.77,
+      lines: 95.76,
```
Comment (jorgemoya) [inline packages/big-design/jest.config.js:15]: This should be going up, not down.

**Comment by bc-apostoliuk** [comment]
@jorgemoya thanks for noticing, I missed pushing this file 😣. I have already pushed. Check it out, please.

**Comment by jorgemoya** [comment]
One other question I have is if this works horizontally or diagonally?

File: packages/big-design/jest.config.js (hunk 1 above)
Comment (bc-apostoliuk) [reply packages/big-design/jest.config.js]: Done)

**Comment by MariaJose** [comment]
Hey Lgtm!  just a question, I tried to select the cells vertically from top to bottom, but noticed that if I want to select them from bottom to top the cells don't get selected. Is this something would be good to be added it? 🍹  or only is gonna be from top to bottom?

**Comment by bc-apostoliuk** [comment]
@MariaJose Hey, for now, it's gonna be from top to bottom.

**Pull request merged**